src/netappfiles-preview/HISTORY.rst

```diff
@@ -3,6 +3,8 @@
 Release History
 ===============
 
+0.2.0
++++++
```

Comment:
There should be something here.

Reply:
I will add something in the next release which is coming in the next couple of weeks.